tests/test_slots.py

```diff
@@ -803,3 +803,21 @@ class NEW_A:
         assert new_a.b == 2
         assert new_a.c == 3
         assert not hasattr(new_a, "d")
+
+
+@pytest.mark.parametrize("cls", [A])
```

Comment:
is this an accident?

Reply:
I copied the above test that does the same… Tried to fit existing patterns, but can scrub this if you'd prefer.

Reply:
Uh yeah and if the above has such fake parametrization could you kill it too? I'm on a phone rn so can't check.

Reply:
Will do. Haha. On phone too. I suppose I'll get the laptop out! 😂

Reply:
Done. Removed from existing tests too in a separate commit up front.